### ISSUE TITLE
WP MYSQL database connection with .env file

### ISSUE THREAD

**Issue by periode** [MPAT-eu/MPAT-core#2]
Hello,

I'm trying to run a local install of MPAT and I'm having issues when wordpress tries to connect to the database.

I'm running on OSX 10.13, MySQL 8.15, serving files with the built-in PHP7 server, and browsing in Firefox 65.

The error I get is:

```
Warning: mysqli_real_connect(): (HY000/2002): No such file or directory in /Users/Pierre/Code/MPAT-core/web/wp/wp-includes/wp-db.php on line 1531
```

I have mysql running on localhost:3306, and I can confirm it by connecting to it with user `mpat` and password `mpat` through SQLPro.

My .env file looks like this:

```
DB_NAME=mpat
DB_USER=mpat
DB_PASSWORD=mpat
DB_HOST=localhost:3306

WP_ENV=development
WP_HOME=http://localhost
WP_SITEURL=${WP_HOME}/wp
WP_DOMAIN=localhost
PATH_CURRENT_SITE=/ # path of WP instance, leave / if configuring wp as virtualhost, otherwise specify path with leading and trailing slashes
WP_MULTISITE=FALSE # if DB is not properly configured, setting MULTISITE to TRUE might lead to fatal errors, follow readme.md for instructions
```

Anything I'm missing? Thanks!

**Comment by jcdufourd** [comment]
I have 
DB_HOST=localhost
(without the :3306)
The rest looks good.

**Comment by periode** [comment]
Thanks! But it still doesn't work when I remove the port number.

When I change `DB_HOST` to `127.0.0.1` I get an error about SHA auth method:

```Warning: mysqli_real_connect(): The server requested authentication method unknown to the client [caching_sha2_password] in /Users/Pierre/Code/MPAT-core/web/wp/wp-includes/wp-db.php on line 1531

Warning: mysqli_real_connect(): (HY000/2054): The server requested authentication method unknown to the client in /Users/Pierre/Code/MPAT-core/web/wp/wp-includes/wp-db.php on line 1531
```

Any thoughts?

**Comment by jcdufourd** [comment]
This is wordpress stuff, not MPAT specific...

**Comment by periode** [comment]
Got it. It was indeed a problem of using different versions of MySQL and PHP. (Solved using MySQL 8.15+ and PHP 7.2+).

Thank you!
